orttraining/orttraining/eager/opgen/opgen/generator.py

```diff
@@ -328,7 +328,30 @@ def _write_function_body(self, writer: writer.SourceWriter, mapped_func: MappedO
                 )
                 writer.writeline()
 
+        last_op_index = len(ctx.ops) - 1
+        equal_onnxop_found = 0
```

Comment:
I think with Python you can use `True` and `False` for boolean data types

Reply:
fixed in last pushed commit (https://github.com/microsoft/onnxruntime/pull/12031/commits/c169b380af3501d4052ca20de0f7ade3ce3c566c)